templates/module/client/rest/tx.go.tmpl

```diff
@@ -1,16 +1,16 @@
 package rest
 
 import (
-	"bytes"
-	"net/http"
+	// "bytes"
+	// "net/http"
 
 	"github.com/gorilla/mux"
 
 	"github.com/cosmos/cosmos-sdk/client/context"
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/types/rest"
-	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
-	"github.com/{{ .User }}/{{ .Repo }}/x/{{ .NameLowerCase }}/types"
+	// sdk "github.com/cosmos/cosmos-sdk/types"
+	// "github.com/cosmos/cosmos-sdk/types/rest"
+	// "github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	// "github.com/{{ .User }}/{{ .Repo }}/x/{{ .NameLowerCase }}/types"
```

Comment:
is this because it removes itself on first save? maybe we add a comment to why they are commented?

Reply:
👍